src/action.ts

```diff
@@ -84,5 +96,15 @@ export async function run(): Promise<void> {
 	// Set the built releaseId in the output
 	core.setOutput('release_id', releaseId);
 
+	if (releaseId) {
+		rawVersion = await balena.getReleaseVersion(parseInt(releaseId, 10));
+	}
+
+	core.setOutput('raw_version', rawVersion);
+
+	if (rawVersion && core.getBooleanInput('create_ref', { required: false })) {
+		await createRef(rawVersion, context.payload.pull_request?.head.sha);
+	}
```

Comment:
This duplicate code is making me realize we should move the code between line 56-75 to the beginning since it doesn't try to build a release. Then, we can just build a release and do this createRef code once...we can do it after this PR though